requirements.yml

```diff
@@ -1,8 +1,2 @@
 ---
-# Upstream is slow to accept pull requests.
```

Comment:
I resemble this remark!

Reply:
I don't think, there's a reason to keep it.

Anyway, i have seen worse.